index.html

```diff
@@ -39,7 +39,75 @@ <h3>LET’S CONNECT</h3>
 
   <section id="works">
     <div class="project">
-      <img src="" alt="">
+      <img class="project_screenshot" src="/images/Snapshoot Portfolio.png" alt="screenshot of Tonic project">
+      <h2 class="project_title"></h2>
+      <p class="project_info">
+        <span class="project_company">CANOPY</span>
+        <span class="project_type">Back End Dev</span>
+        <span class="project_year">2015</span>
+      </p>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups
+        required.</p>
+      <ul class="project_languages">
+        <li class="language">html</li>
+        <li class="language">css</li>
+        <li class="language">javaScript</li>
+      </ul>
+      <button class="see_project_btn">See Project</button>
```

Comment:
Hey @xtrahuman, thank you for the review. However, the images are appearing correctly on my end and I also checked the links and image names and they do match. Any suggestions on what's going wrong?
![image](https://user-images.githubusercontent.com/50721479/158878271-ec7e6752-9b18-4435-bec4-11a86ed3d986.png)

Reply:
@xtrahuman I think the problem is the live preview. I googled it and many other people had the same issue.

Reply:
Hello @xtrahuman, I fixed the issue. Turned out the path to the images was the problem (a "." was missing since on GitHub live preview my code and my images are in 2 different folders). Thanks.